k8s/cloud/dev/kustomization.yaml

```diff
@@ -1,35 +1,37 @@
 ---
 apiVersion: kustomize.config.k8s.io/v1beta1
 kind: Kustomization
-commonLabels:
-  app: pl-cloud
 namespace: plc-dev
 replicas:
-- name: api-server
-  count: 1
-- name: auth-server
-  count: 1
-- name: profile-server
-  count: 1
-- name: cloud-proxy
-  count: 1
-- name: project-manager-server
-  count: 1
-- name: vzmgr-server
-  count: 1
-- name: scriptmgr-server
-  count: 1
+- count: 1
+  name: api-server
+- count: 1
+  name: auth-server
+- count: 1
+  name: profile-server
+- count: 1
+  name: cloud-proxy
+- count: 1
+  name: project-manager-server
+- count: 1
+  name: vzmgr-server
+- count: 1
+  name: scriptmgr-server
 resources:
 - ../base
 - ../overlays/exposed_services_ilb
 - plugin_db_updater_job.yaml
-patchesStrategicMerge:
 # bq_config is useful for testing, but we don't want dev clusters to typically send data to bq.
 # - bq_config.yaml
```

Comment:
move this section down into patches
(you'll also want to rewrite it as `- path: bq_config.yaml`)

Reply:
Nice catch. Missed this